manifests/attr.pp

```diff
@@ -1,9 +1,10 @@
 # Install FreeRADIUS config snippets
 define freeradius::attr (
   String $source,
-  Freeradius::Ensure $ensure = present,
-  Optional[String] $key      = 'User-Name',
-  Optional[String] $prefix   = 'filter',
+  Freeradius::Ensure $ensure           = present,
+  Optional[String] $key                = 'User-Name',
+  Optional[String] $prefix             = 'filter',
+  Optional[Enum['yes', 'no']] $relaxed = 'no',
```

Comment:
Given this is optional, it should probably default to undef - otherwise it should just be Enum type. My strong preference is for this to default to undef - as FreeRADIUS allows it to be omitted and follow a default in FR, so we should too.

(Yes lines 5 and 6 are also not following this pattern, but, better not build more technical debt! For those, we should remove Optional, but that can be another clean up PR.)

Reply:
That's been updated in my repo now.